### PR TITLE
Resolve type errors: Quick Cleanup

### PR DESCRIPTION
**Pull request by CharlieS1103**

### CODE REVIEW

File: src/types/spicetify.d.ts [hunk 1]
```diff
@@ -1997,10 +1997,12 @@ declare namespace Spicetify {
...
 	namespace Config {
 		const version: string;
-		const current_theme: string;
-		const color_scheme: string;
+		let current_theme: string;
+		let color_scheme: string;
 		const extensions: string[];
 		const custom_apps: string[];
+		let local_theme : string;
+		let local_color_scheme : string;
```
Comment (rxri) [inline src/types/spicetify.d.ts:2005]: why do you directly modify the spicetify types, don't do it please

Comment (SunsetTechuila) [reply src/types/spicetify.d.ts:2005]: Or it is better to modify the types in the spicetify cli repo

Comment (rxri) [reply src/types/spicetify.d.ts:2005]: Well, technically the `Config` object shouldn't be used for marketplace usage. It should be in its own object that only marketplace uses

Comment (SunsetTechuila) [reply src/types/spicetify.d.ts:2005]: I think it was done this way to keep compatibility with extensions that check current theme and themes that check current color scheme

Comment (rxri) [reply src/types/spicetify.d.ts:2005]: but they don't check the `local_theme` nor `local_color_scheme`, so this stuff could be kept in local storage for example

Comment (SunsetTechuila) [reply src/types/spicetify.d.ts:2005]: Yes, `local_theme` and `local_color_scheme` definitely shouldn't be a part of the spicetify types declaration file

File: src/components/Modals/BackupModal/index.tsx [hunk 2]
```diff
@@ -56,6 +56,7 @@ const BackupModal = () => {
    * Prompt user to select a file to import and then run importMarketplace
    */
   const importSettingsFromFile = async () => {
+  // @ts-expect-error: `showOpenFilePicker not defined in type Window`
```
Comment (theRealPadster) [inline src/components/Modals/BackupModal/index.tsx:59]: This one could probably be fixed somehow...

Comment (SunsetTechuila) [reply src/components/Modals/BackupModal/index.tsx:59]: `window["showOpenFilePicker"]()`

Comment (rxri) [reply src/components/Modals/BackupModal/index.tsx:59]: In this case, the window type should be extended, and definitely not resolved like @SunsetTechuila suggested

Comment (SunsetTechuila) [reply src/components/Modals/BackupModal/index.tsx:59]: Then https://www.npmjs.com/package/@types/wicg-file-system-access

Comment (theRealPadster) [reply src/components/Modals/BackupModal/index.tsx:59]: Yeah, if that package adds the type for it, we should probably add that 👍

**Review by theRealPadster — CHANGES_REQUESTED** [review]
Please add that types package to resolve the `"showOpenFilePicker" not defined in type Window` error

**Review by rxri — CHANGES_REQUESTED** [review]
This still should be done before merging, https://github.com/spicetify/spicetify-marketplace/pull/672#discussion_r1428481530

**Comment by rxri** [comment]
No activity and none of requested changes have been implemented

**Pull request closed without merging**